src/classes/ADDR_Account_TDTM.cls

```diff
@@ -48,123 +51,122 @@ public class ADDR_Account_TDTM extends TDTM_Runnable {
-           // only test StateCode if picklists enabled.
-           (ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled && acc1.get('BillingStateCode') != null);
-
-        return isSpecified;   
-    }
-
-    /*******************************************************************************************************
-    * @description utility to compare two Accounts' (billing) addresses
-    * @param acc1 an Account
-    * @param acc2 an Account
-    * @return boolean. true if there are any changes between the two accounts.
-    ********************************************************************************************************/
-    private boolean isAccountAddressChanged(Account acc1, Account acc2) {
-        if (acc1 == null || acc2 == null) 
-           return false;
+        // handle accs with address info removed
+        if(ListAccsAddrInfoCleared.size() > 0)
+            addrInfoDeleted(ListAccsAddrInfoCleared, dmlWrapper);
         
-        boolean isChanged = 
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingStreet, acc2.BillingStreet) ||
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingCity, acc2.BillingCity) ||
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingState, acc2.BillingState) ||
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingPostalCode, acc2.BillingPostalCode) ||
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingCountry, acc2.BillingCountry);
-
-        if (!isChanged && ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled) {
-            isChanged = 
-                !ADDR_Addresses_TDTM.equalsCaseSensitive(string.valueOf(acc1.get('BillingStateCode')), string.valueOf(acc2.get('BillingStateCode'))) || 
-                !ADDR_Addresses_TDTM.equalsCaseSensitive(string.valueOf(acc1.get('BillingCountryCode')), string.valueOf(acc2.get('BillingCountryCode')));
-        }
-        UTIL_Debug.debug('****Address changed: ' + isChanged);
-        return isChanged;   
-
+        hasRunAddrTrigger = true;
+                            
+        return dmlWrapper;    
     }
-
+    
     /*******************************************************************************************************
     * @description for each Account, create a new default address and add it to dmlWrapper
-    * @param listAcc a list of Accounts
+    * @param ListAcc a List of Accounts
     * @param dmlWrapper to hold the Addresses that need creating
     * @return void
     ********************************************************************************************************/
-    private void createAddrFromAcc(list<Account> listAcc, DmlWrapper dmlWrapper) {
-        list<Address__c> listAddr = new list<Address__c>();
-        for (Account acc : listAcc) {
+    private void createAddrFromAcc(List<Account> ListAcc, DmlWrapper dmlWrapper) {
+        List<Address__c> ListAddr = new List<Address__c>();
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.